scenarios/pfcp-agent-failure.xml

```diff
@@ -24,14 +27,25 @@
         <step name="AF-Check-Flows-2" requires="^"
               exec="onos-cli-grep ${OCI} up4:read-flows 'UL flows=0, DL flows=0'"/>
 
+        <step name="AF-Mid-Check-Interfaces" requires="^"
+              exec="onos-cmd ${OCI} 'up4:read-entities -i -g' 'tmp/Mid_check_interfaces.log'"/>
+
         <!-- Reboot SMF because it probably doesn't react well to failures -->
         <group name="AF-Restart-Smf" requires="~AF-Check-Flows-2">
-            <step name="AF-Kill-Smf"
-                  exec="mock-smf-cmd kill"/>
-            <step name="AF-Start-Smf" requires="^"
-                  exec="mock-smf-cmd start"/>
-            <step name="AF-Associate-Smf" requires="^" delay="1"
-                  exec="mock-smf-cmd associate"/>
+            <step name="AF-Kill-Smf" delay="1"
+                  exec="${DOCKER_COMPOSE_CMD} restart mock-smf"/>
+            <step name="AF-Configure-Smf" requires="^"
+                  exec="${DOCKER_COMPOSE_CMD} exec -T mock-smf pfcpctl -c configure --n3-addr 140.0.0.1 --remote-peer pfcp-agent"/>
+            <step name="AF-Associate-Smf" requires="^"
+                  exec="${DOCKER_COMPOSE_CMD} exec -T mock-smf pfcpctl -c associate"/>
         </group>
+
+        <step name="AF-Post-Check-Interfaces" requires="AF-Restart-Smf"
+              exec="onos-cmd ${OCI} 'up4:read-entities -i -g' 'tmp/Post_check_interfaces.log'"/>
+
+        <step name="AF-Check-Up4-N3-Interface" requires="AF-Restart-Smf"
+              exec="onos-cli-grep ${OCI} up4:read-interfaces 'UpfInterface(type=ACCESS, prefix=140.0.0.1/32'"/>
+        <step name="AF-Check-Up4-Uepool-Interface" requires="AF-Restart-Smf"
+              exec="onos-cli-grep ${OCI} up4:read-interfaces 'UpfInterface(type=CORE, prefix=17.0.0.0/16'"/>
```

Comment:
This check can stay, and should help us quickly verify the restart was successful.